theme/keyframes.ts

```diff
@@ -14,4 +14,34 @@ export const keyframes: CssKeyframes = {
       transform: 'translateY(0)',
     },
   },
+  slideRightAndFade: {
```

Comment:
I just put up a PR in the mono repo, so I can add these there so you can remove them. I moved the other one.

Reply:
https://github.com/leather-wallet/mono/pull/37